spec/models/invoice_spec.rb

```diff
@@ -19,4 +19,9 @@
     @invoice = create(:invoice, customer_id: @customer.id)
   end
 
+  describe "instance methods" do
+    it "#formatted_time" do
+      expect(@invoice.formatted_time).to eq(@invoice.created_at.to_datetime.strftime("%A, %B %d, %Y"))
+    end
+  end
```

Comment:
describe "instance methods" do
    describe "#formatted_time" do
         it "formats the date and time" do

Reply:
I'll fix this in the next PR I make, thank you for noticing